examples/lookup/main.go

```diff
@@ -56,6 +56,9 @@ func run() error {
 
 func RandomString() string {
 	buf := make([]byte, 6)
-	rand.Read(buf)
+	_, err := rand.Read(buf)
```

Comment:
let's just ignore the error. This is example code made to be easy to read. error handling such as it is can be omitted.